File: src/1217.minimum-cost-to-move-chips-to-the-same-position.1329/solution.ts
function minCostToMoveChips (position: number[]): number {
    let odd = 0;
    for (let i = 0; i < position.length; i++) {
        if (position[i] & 1) {
            odd++;
        }
    }
    return Math.min(odd, position.length - odd);
}

File: src/1217.minimum-cost-to-move-chips-to-the-same-position.1329/solution.java
class Solution {
    public int minCostToMoveChips(int[] position) {
        int odd = 0;
        int even = 0;
        for (int pos : position) {
            if (pos % 2 == 0) {
                even++;
            } else {
                odd++;
            }
        }
        return Math.min(odd, even);
    }
}